third_party/libevent/event.c

```diff
@@ -65,8 +62,6 @@
 #include <string.h>
 #include <assert.h>
 #include <time.h>
-#endif  // STARBOARD
-
```

Comment:
This empty line is not ours:
https://chromium.googlesource.com/chromium/src/+/70.0.3538.124/base/third_party/libevent/event.c#53